examples/offloader.js

```diff
@@ -31,53 +33,54 @@ if (fs.existsSync('polldata.json')) {
 
 client.logOn(logOnOptions);
 
-client.on('loggedOn', function() {
-	console.log("Logged into Steam");
+client.on('loggedOn', function () {
+	console.log('Logged into Steam');
 });
 
-client.on('webSession', function(sessionID, cookies) {
-	manager.setCookies(cookies, function(err) {
```

Comment:
@DoctorMcKay In all the cases where I renamed it, there is one function with a error-first callback, and inside it there's another function that is called with error-first callback, as a result, there are 2 objects named `err`. ESLint is also complaining about it, and I think it's right. I've changed it to be more clear.

Example (of why I did it):

```
function1(some, arguments, function (err, result1) {
    function2(another, arguments, (err, result2) {
        // there is a unclearness, because err was defined twice.
    }
}
```